src/classes/speciesSite.h

```diff
@@ -25,12 +25,15 @@ class SpeciesSite : public Serialisable<CoreData &>
 {
     public:
     // Site Type
-    enum class SiteType
+    enum SiteType
     {
         Static,  /* Site is based on fixed atom indices within the species */
         Dynamic, /* Site is atomic and based on elements and atom types */
         Fragment /* Site is based on a NETA description */
     };
+    // EnumOptions for SiteType
+    static EnumOptions<SiteType> types();
```

Comment:
To be consistent with the rest of the code:
```suggestion
    static EnumOptions<SiteType> siteTypes();
```